core/server/data/importer/importers/data/settings.js

```diff
@@ -17,6 +18,17 @@ const deprecatedSupportedSettingsMap = {
     ghost_foot: 'codeinjection_foot'
 };
 
+const deprecatedSupportedSettingsOneToManyMap = {
```

Comment:
I like the generalised approach of this! Though I wonder if it might be a little easier to follow with an object like:
```js
unsplash: [{
  getValue(valueFromDb) {
    return JSON.parse(valueFromDb).isActive;
  },
  key: 'unsplash',
  group: 'unsplash',
  type: 'boolean'
]]
```

Reply:
I like this notation! It's smaller and agree, it's way more readable :+1:

Reply:
Tried playing with it a little, but having multiple fields (slack has multiple mappings as well) then there's lots of duplicate code that comes from having to do error handling for `JSON.parse`. Might come up with something nice later on, but for now leaving it as "good enough, does the job" code :sweat_smile:

Reply:
Makes perfect sense to me :relaxed: